src/interfaces/IMetaMorpho.sol

```diff
@@ -174,6 +172,8 @@ interface IMetaMorphoBase {
     /// - Withdrawals from markets that are expected to be withdrawn from during reallocation.
     /// @dev Sender is expected to pass `assets = type(uint256).max` with the last MarketAllocation of `allocations` to
     /// supply all the remaining withdrawn liquidity, which would ensure that `totalWithdrawn` = `totalSupplied`.
+    /// @dev A supply in a reallocation step will make the reallocation revert if the amount is greater than the amount
+    /// extracted (i.e. total withdrawn minus total supplied) in the previous steps.
```

Comment:
```suggestion
    /// extracted from the previous steps and remaining (i.e. total withdrawn minus total supplied).
```

Reply:
[alternative](https://github.com/morpho-org/metamorpho/pull/420/files/1cd6a0325b3691904056b446e2efa9d201e6daf7#r1633490943)

Reply:
I went with the alternative